packages/js/product-editor/src/blocks/product-fields/downloads/edit-downloads-modal/style.scss

```diff
@@ -25,9 +25,11 @@ $gutenberg-blue: var(--wp-admin-theme-color);
 		cursor: pointer;
 	}
 
-	.woocommerce-image-gallery, .components-form-file-upload {
+	.woocommerce-image-gallery,
+	.components-form-file-upload {
 		display: inline-block;
 		vertical-align: middle;
+		padding-left: $gap;
```

Comment:
Adding the padding here means its also being applied to the `woocommerce-image-gallery` and thus causing the image to also move to the right, could we only add the padding to the `components-form-file-upload`?
<img width="305" alt="Screenshot 2023-11-03 at 1 17 57 PM" src="https://github.com/woocommerce/woocommerce/assets/2240960/f0a58b68-8a15-4707-a318-e03adfc25882">